src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -497,6 +499,10 @@ protected void writeOutResults(String outputParentDirName) throws IOException {
     // write the test run summary to a file
     objectWriter.writeValue(runSummaryFile, summary);
     logger.info("Test run summary written to file: {}", runSummaryFile.getName());
+
+    // Write the MCTerra Component versions of target environment to a file
+    objectWriter.writeValue(terraVersionFile, KubernetesClientUtils.getComponentVersions());
+    logger.info("MCTerra Component versions written to file: {}", terraVersionFile.getName());
```

Comment:
If the service is not deployed to a Kubernetes cluster, then this will just generate an empty file?

I think longer-term we need to allow different ways of fetching the version, so that we can get a meaningful version string in different ways (e.g. git hash for local server). This is okay for the first pass, but let's make a ticket to allow different ways of fetching the version.

Reply:
Correct! As discussed in perf framework meeting, the component version framework only supports services deployed to Kubernetes. 

I created QA ticket `https://broadworkbench.atlassian.net/browse/QA-1445` to track this conversation.